AWS_FOR_FLUENT_BIT_VERSION

```diff
@@ -1 +1 @@
-2.31.7
\ No newline at end of file
+2.28.5
```

Comment:
Should this be the stable file?

Reply:
We've agreed we aren't changing stable. This file must container whatever version the pipeline is currently releasing and the sync tasks are currently sync-ing. So that's why.